### PR TITLE
CRDCDH-1028 Redesign Submission Flow

### PR DESCRIPTION
**Pull request by amattu2**
### Overview

This PR implements the redesigned Submission Flow (no ticket) and adds the submission guide adornment to the top of the section.

NOTE: Ignore the fact that the steps go from 1 to 3, step 2 will be added later. Also ignore the verbiage differences in the mockup vs current implementation. The implemented text is based on existing requirements.

### Change Details (Specifics)

- Add user guide PDF and link to the top of the submission flow
- Rename "DataSubmissionUpload" to just "DataUpload"
- Refactor the ValidationControls design from CSS classes to Emotion Styled 
- Refactor FlowWrapper to support new design

### Related Ticket(s)

CRDCDH-1028 – Actual story
CRDCDH-1001 – Design reference

### CODE REVIEW

File: src/components/DataSubmissions/DataUpload.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ const StyledMetadataText = styled(StyledUploadTypeText)(() => ({
 
 const StyledUploadFilesButton = styled(Button)(() => ({
   minWidth: "137px",
-  minHeight: "44px",
+  height: "44px",
```
Comment (Alejandro-Vega) [inline src/components/DataSubmissions/DataUpload.tsx:30]: For the other buttons I just removed the height property and made sure the padding is 10px and lineHeight is 24px to match the Figma. The 1px border will increase the height to 46px. (All other buttons are 46px) Let me know if you agree, or if we should modify the heights of the other buttons to a fixed 44px.

**Review by Alejandro-Vega — CHANGES_REQUESTED** [review]
Left a small comment on the buttons sizes. The rest LGTM.

File: src/components/DataSubmissions/DataUpload.tsx (hunk 1 above)
Comment (amattu2) [reply src/components/DataSubmissions/DataUpload.tsx]: Agreed. I reverted the height override so the buttons are all consistent. 66b8644

**Review by Alejandro-Vega — APPROVED** [review]
LGTM!

**Pull request merged**